support/CMakeLists.txt

```diff
@@ -152,3 +152,15 @@ add_custom_target(uninstall-rootfs-image-module
   COMMAND ${CMAKE_COMMAND} -D CMAKE_INSTALL_COMPONENT=rootfs-image-module -P ${CMAKE_BINARY_DIR}/cmake_uninstall.cmake
   WORKING_DIRECTORY ${CMAKE_BINARY_DIR}
 )
+
+install(FILES ${CMAKE_SOURCE_DIR}/support/version
```

Comment:
I can't see that this file exists in `support/version`. Did you forget to submit or generate it?